README.md

```diff
@@ -152,6 +152,55 @@ using (IDbConnection conn = new SnowflakeDbConnection())
 }
 ```
 
+<a id="sample-connection-strings"></a>
+#### Using Special Characters in Connection Passwords
+
+The .NET connector now uses Microsoft [DbConnectionStringBuilder](https://learn.microsoft.com/en-us/dotnet/api/system.data.oledb.oledbconnection.connectionstring?view=dotnet-plat-ext-6.0#remarks) to following .NET specification for escaping characters in connection strings. 
+
+The following examples show how you can include different types of special characters in a connection string:
+
+- To include a single quote (') character:
+
+  ``` cs
+  string connectionString = String.Format(
+    "account=testaccount; " +
+    "user=testuser; " +
+    "password=test'password;"
+  );
+  ```
+
+- To include a double quote (") character:
+
+  ``` cs
+  string connectionString = String.Format(
+    "account=testaccount; " +
+    "user=testuser; " +
+    "password=test\"password;"
+  );
+  ```
+
+- To include a semicolon (;):
+
+  ``` cs
+  string connectionString = String.Format(
+    "account=testaccount; " +
+    "user=testuser; " +
+    "password=\"test;password\";"
+  );
+  ```
+
+- To include an equal sign (=):
+
+  ``` cs
+  string connectionString = String.Format(
+    "account=testaccount; " +
+    "user=testuser; " +
+    "password=test=password;"
+  );
+  ```
+
+  Note that previously you needed to use a double equal sign (==) to escape the character, but it is no longer required.
```

Comment:
What does "it" refer to?
Should this be "Prior to version n, you needed to use two equal signs (==) to specify an equal sign in the password. In version n and more recent versions, you no longer need to use two equal signs."

Reply:
Note that previously you needed to use a double equal sign (==) to escape the character. However, beginning with version 2.0.18, you can use a single equal size.